tests/input/null_coalesce_operator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+$foo = isset($_GET['foo']) ? $_GET['foo'] : 'foo';
+
+$bar = isset($bar) ? $bar : 'bar';
+
+$bar = isset($bar['baz']) ? $bar['baz'] : 'baz';
+
+if (isset($foo)) {
```

Comment:
@kukulich This case is already covered? I guess tests are failing here

Reply:
Nope, it's not. It's a lot more complicated than simple ternary operator.